seed/challenges/05-apis-and-microservices/basic-node-and-express.json

```diff
@@ -128,9 +128,9 @@
       "id": "587d7fb1367417b2b2512bf1",
       "title": "Serve JSON on a specific route",
       "description": [
-        "While an HTML server serves (you guessed it ?) HTML, an API serves data. A REST API allows data exchange in a simple way, without the need for clients to know any detail about the server. The client only needs to know where the resource is (the URL), and the action it wants to perform on it (the verb). The GET verb is used when you are fetching some information, without modifying anything. In these days the preferred data format to move these informations around the web is JSON. In few words JSON is a convenient way to represent a JavaScript object as a string, so it can be easily transmitted.",
-        "Let’s create a route that responds with JSON at the path (/json). You can do it as usual, with the app.get() method. Inside the route handler use the method res.json(<object>). This method closes the request-response loop, returning the data. Behind the scenes it converts a valid Javascript object into a string. Then sets the appropriate headers to tell your browser that you are serving JSON, and sends the data back. A valid object has the usual structure {key: data}. data can ba a number, a string, a nested objects or an array. data can also be a variable or the result of a function call. It will be evaluated before converting into a string.",
-        "Serve the object <code>{message: 'json'}</code> as a response in JSON format, to the GET requests to the route /json. Point your browser to <your-app-url>/json, you should see the message on the screen."
+        "While an HTML server serves (you guessed it ?) HTML, an API serves data. A REST API allows data exchange in a simple way, without the need for clients to know any detail about the server. The client only needs to know where the resource is (the URL), and the action it wants to perform on it (the verb). The GET verb is used when you are fetching some information, without modifying anything. These days, the preferred data format for moving information around the web is JSON. In a few words, JSON is a convenient way to represent a JavaScript object as a string, so it can be easily transmitted.",
```

Comment:
@Greenheart I also see a couple other things that could use improvement and I'd like to see if you agree. I kind of did this PR in a hurry last night, so didn't have time to think too critically about this.

**First:** 
> While an HTML server serves (you guessed it ?) HTML...

Not sure if I like the "?" here, I think it should maybe be:

> While an HTML server serves (you guessed it!) HTML...

**Second:**

The reference to APIs in the first paragraph here is overall a bit confusing. We are provided with some good info on APIs, but I don't think the concept is sufficiently tied in to the challenge at hand. For instance, as a first time learner of this info (which gives me a good perspective), I'm not totally clear on what part of what we're doing in this challenge actually involves and API.

Ae we actually creating an API that serves JSON with this code? Or are we hitting an API that responds with JSON? Or both? I think the former, but if so, who / what else is going to utilize this API down the line? 

I think it's exemplifies the concept of serving JSON well, but could perhaps tie in to the larger scope of building a web app better. Ideally in a concise way that would only add another sentence or 2. Any thoughts?

**Third:**
> Serve the object <code>{message: 'Hello json'}</code> as a response in JSON format

Since we tell campers in the NPM challenges that JSON keys and values must always be wrapped in double quotes, do you think we should point out here that its best practice not to wrap keys in double quotes like this when sending JSON? I noticed the example follows just regular object notation, and I think someone mentioned to me that this is best practices.